File: PRAKTIKUM 4/Bahasa C/PRAK405-2510817120014-NailaAtresiaRahma.c
#include <stdio.h>

int main(){

    int n,kelipatan, hasil,hasil2,total;
    hasil = hasil2 = total = 0;
    scanf("%d %d", &n, &kelipatan);
    for(int i=1;i<=n;i++) {
        for(int j=i;j>=1;j--) {
            printf("(%d * %d) ",j,kelipatan);
            if (j>1){
                printf(" + ");
    }
    }
        hasil = (i*kelipatan);
        hasil2 += hasil;
        printf (" = %d",hasil2);
        total+= hasil2;
        printf("\n");
    }
    printf("%d",total);
    return 0;
}

File: PRAKTIKUM 4/Bahasa Python/PRAK405-2510817120014-NailaAtresiaRahma.py
n,kelipatan = input().split(); n = int(n); kelipatan = int(kelipatan)

hasil1 = 0; hasil2 = 0; total = 0
i = 0
while i < n :
    i += 1
    j = i
    if i == 0:
        break
    while j > 0 :
        j -= 1
        print("(%d * %d)" %(j+1,kelipatan), end=" ")
        if(j > 0) :
            print(" + ", end=" ")
    hl = i * kelipatan
    hasil2 += hl
    print("= %d" %(hasil2))
    total += hasil2
print ("%d" %(total))